communities.json

```diff
@@ -679,6 +679,14 @@
       "quote": "Discord both strengthened our community and dramatically improved discussion amongst the developers.",
       "inviteCode": "G7Qnnhy",
       "githubUrl": "https://github.com/nestjs"
+    },
+    {
+      "logo": "codetogether.svg",
+      "title": "code::together",
+      "quote": "code::together is an online Discord community of programmers and developers.",
+      "quoteSourceUrl": "https://together.codes/",
+      "inviteCode": "https://discord.gg/HPK3hB6",
```

Comment:
This invite code is supposed to be the string **after** discord.gg, so it should be just "HPK3hB6" in this case.

Reply:
Oh sorry tahnks for the change :)